src/main/scala/com/microsoft/hyperspace/Hyperspace.scala

```diff
@@ -89,6 +89,20 @@ class Hyperspace(spark: SparkSession) {
     indexManager.refresh(indexName, mode)
   }
 
+  /**
+   * Optmizes index files by merging smaller files into larger files.
+   *
+   * Note: This api does NOT refresh (i.e. update) the index if the underlying data changes. It
+   * only rearranges the index data into a better layout, by compacting small index files.
+   *
+   * @param indexName Name of the index to optimize.
+   * @param mode Index optimization mode. "quick" refers to optimization of only small index
+   *             files, based on a threshold. "full" refers to recreation of index.
+   */
+  def optimizeIndex(indexName: String, mode: String = OPTIMIZE_MODE_QUICK): Unit = {
```

Comment:
Can we follow the same pattern as `refreshIndex` for exposing default param? (It's easy for language binding especially for C# - e.g, no need to copy default value, etc.)

Reply:
thanks @imback82 , could you please check the latest commit if I understood correctly?

Reply:
Yes. Can you remove the default value now?

Reply:
thanks @imback82 , updated